CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
-## Unreleased
-
+## 0.82.0 / 2025-04-17
+
+* [CHANGE] Add more API validations to the ScrapeConfig CRD. #7413
+* [FEATURE] Add RemoteWrite support for ThanosRuler CRD. #7444 #7455
+* [FEATURE] Add `msTeamsV2Configs` receiver to AlertManagerConfig CRD. #7464
+* [FEATURE] Add `content`, `username` and `avatarURL` fields to `discordConfigs` receiver in AlertmanagerConfig CRD. #7307
+* [FEATURE] Add `convertClassicHistogramsToNHCB` field to `ServiceMonitor`, `PodMonitor`, `Probe` and `ScrapeConfig`. #7448
+* [FEATURE] Add `timeout` field to `webhookConfigs` receiver in AlertManagerConfig CRD. #7310
+* [FEATURE] Add `retain` field to Prometheus CRD. #7377
```

Comment:
Right now the field has no effect so I'd avoid mentioning it in the changelog for now.
```suggestion
```